### ISSUE TITLE
Allow to duplicate Rubric items

### ISSUE THREAD

**Issue by moveisi** [PrairieLearn/PrairieLearn#10259]
When making rubrics, it is helpful to easily duplicate Rubric items and modify them when items are very similar.
(Also see [discussion on pl-help](https://prairielearn.slack.com/archives/C266KEH9A/p1721372271679939))

**Comment by jonatanschroeder** [comment]
Duplicate of #9616.

**Comment by nwalters512** [comment]
@jonatanschroeder I don't believe this is actually a duplicate. The linked issue is about copying entire rubrics between questions, but I think this is about copying items _within_ a question's rubric.

**Comment by jonatanschroeder** [comment]
Ok, @moveisi can you expand on what you mean?
